docs/source/traitsui_user_manual/testing/discussions/compatibility_pyface_testing.rst

```diff
@@ -0,0 +1,67 @@
+Compatibility with Pyface test utilities
+========================================
+
+|UITester| is intended to be compatible with Pyface's |ModalDialogTester|,
```

Comment:
The `|ModalDialogTester|` is not working as a link to the pyface docs for me in the built documentation.

I ran:
```
python etstool.py install
python etstool.py docs
python -m http.server
```

Reply:
Thank you for checking, and sorry I should have mentioned that this is expected. Pyface documentation does not include toolkit specific objects.

I kept the cross reference here despite the missing link because there is meant to be a cross reference.  
When that pyface issue is fixed (if ever), the hope is that the cross link here should work (when we generate the documentation again).